### PR TITLE
[MAINT] Update of package install features and coding contributions

### PR DESCRIPTION
**Pull request by josephmje**

### CODE REVIEW

**Comment by josephmje** [comment]
This is still a work in progress. I'll send another alert once it's ready for review.

**Comment by josephmje** [comment]
NOTE: `versioneer.py` and `datman/_version.py` are automatically generated by versioneer and don't need to be reviewed.

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,112 @@
 [metadata]
-description-file = README.md
+url = https://github.com/tigrlab/datman
+author = The TIGRLab
+author_email = dawn.smith@camh.ca
+maintainer = Dawn Smith
+maintainer_email = dawn.smith@camh.ca
+description = datman is a.
```
Comment (DESm1th) [inline setup.cfg]: Miiiight be good to put a real description here. And as we discussed to replace the author and maintainer emails with our lab gmail

Comment (josephmje) [reply setup.cfg]: hahhah thanks for catching that!

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      "name": "Joseph, Michael J",
+      "orcid": "0000-0002-0068-230X"
+    },
+    {
+      "affiliation": "The Centre for Addiction and Mental Health",
+      "name": "Manogaran, Mathuvanthi"
+    },
+    {
+      "affiliation": "The Centre for Addiction and Mental Health",
+      "name": "Herman, Gabrielle"
```
Comment (gabiherman) [inline .zenodo.json]: my orcid is 0000-0003-4695-5251 :)

File: setup.cfg (hunk 1 above)
Comment (gabiherman) [reply setup.cfg]: It makes sense to me to replace author_email with tigrlab email and leave maintainer email as dawn's email -- but I missed the discussion, you may have gone over this

**Pull request merged**